client/my-sites/domains/domain-management/dns/a-record.jsx

```diff
@@ -23,19 +23,20 @@ class ARecord extends Component {
 		const isDataValid = isValid( 'data' );
 		const isAaaaRecord = fieldValues.type === 'AAAA';
 
-		let namePlaceholder = translate( 'Enter subdomain (optional)', {
-			context: 'Placeholder shown when entering the optional subdomain part of a new DNS record',
+		let label = translate( 'Name (optional)', { context: 'DNS record' } );
+		if ( isAaaaRecord ) {
+			label = translate( 'Name (required)', { context: 'DNS record' } );
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **89** times:
`translate( 'Name (required)' )` *ES Score: 11*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/add-dns-record-page)


:information_source: This string already exists without a **context**. Only add a context if the meaning of the string is very specific.